packages/suite/src/actions/wallet/send/sendFormEthereumActions.ts

```diff
@@ -149,6 +149,13 @@ export const composeTransaction =
             }
         } else {
             // TODO: catch error from blockbook/geth (invalid contract, not enough balance...)
+            if (estimatedFee.payload.error) return;
```

Comment:
This is as @mroz22 said not needed. If response was not successful then we always want to show this toast message.